src/test/util_tests.cpp

```diff
@@ -329,23 +329,23 @@ BOOST_AUTO_TEST_CASE(util_VerifyIsLockTimeWithin14days)
 {
     int64_t now = 1494060475;
     int64_t twoWeeksInSeconds = 1209600;
+    int64_t time = now - twoWeeksInSeconds;
 
-    SetMockTime(now);
```

Comment:
I'm not familiar with this code, but since `now` is hard coded to a value, I'm guessing `SetMockTime(now)` was needed.  It is still included on the next test updated below, so was it accidentally removed here?

Reply:
The reference time is injected instead of being hard coded now so the mock time is not needed anymore.